bundles/org.openhab.binding.boschshc/pom.xml

```diff
@@ -1,23 +1,33 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-  <modelVersion>4.0.0</modelVersion>
+	<modelVersion>4.0.0</modelVersion>
 
-  <parent>
-    <groupId>org.openhab.addons.bundles</groupId>
-    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
-    <version>2.5.1-SNAPSHOT</version>
-  </parent>
+	<parent>
+		<groupId>org.openhab.addons.bundles</groupId>
+		<artifactId>org.openhab.addons.reactor.bundles</artifactId>
+		<version>2.5.1-SNAPSHOT</version>
+	</parent>
 
-  <artifactId>org.openhab.binding.boschshc</artifactId>
+	<artifactId>org.openhab.binding.boschshc</artifactId>
+
+	<name>openHAB Add-ons :: Bundles :: BoschSHC Binding</name>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.bouncycastle</groupId>
+			<artifactId>bcpkix-jdk15on</artifactId>
+			<version>1.54</version>
+			<scope>compile</scope>
```

Comment:
Whaaaaat. Is that all the magic? :mage: 

I had excatly the same thing, minus the scope and it didn't compile for me any more!

Excellent work, I'm so happy you are on the team! :-D

Reply:
That's Maven 😞 

In the meantime I took some time to read some pages of the [OpenHAB Developer Guide](https://www.openhab.org/docs/developer/) and found the [Build System Appendix](https://www.openhab.org/docs/developer/buildsystem.html) telling to use ``provided``.

Reply:
Latest update from today:
plugin not yet working with ``provided``! My current manual installation of this jar and the two bouncycastle dependencies in addons folder leads to a runtime error during pairing.

Reply:
Final result regarding  maven "provided" scope:
I was not able to get our add-on working according to the "Build System Appendix, but I found other add-on like the ``org.openhab.binding.amazondashbutton`` using also the ``compile`` scope. I will use our working solution and wait until we get some feedback later on.